public-ui/src/app/data-browser/views/fmh-view/fmh-view.component.html

```diff
@@ -1,6 +1,6 @@
 <hr>
 <div class="topic-display">
-  <div class="topic-body-default">
+  <div class="topic-body-default" *ngIf="conditionQuestions.length > 0">
```

Comment:
Display the metadata text only if the condition questions / family member questions are available.